packages/valdueza-storybook/.storybook/config.js

```diff
@@ -1,7 +1,20 @@
-import { configure } from '@storybook/react';
+import { addParameters, configure } from '@storybook/react';
 //https://github.com/storybooks/storybook/tree/next/addons/storyshots/storyshots-core#configure-jest-to-work-with-webpacks-requirecontext ("Macro" subsection)
 import requireContext from 'require-context.macro';
 
+const newViewports = {
+  chromeExtension: {
+    name: 'extension',
+    styles: {
+      width: '350px',
+      height: '500px',
```

Comment:
Maybe we can get those values from `sanes-chrome-extension/src/theme/constants.ts`?